### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Eldonratzlaff**
Add redux to proyect

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const BooksSlice = createSlice({
+    name: 'Books',
+    initialState,
+    reducers: { 
+      addBook: (state, action) => {
+        state.Books.push(action.payload);
+      }         
+    }
+});
```
Comment (alexander16108) [inline src/redux/books/booksSlice.js]: - as required you should create 2 reducers for both the add book and remove book but as we can see you have only one
- you are to export your reducer add book and remove book as actions

File: src/redux/categories/categoriesSlice.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    },      
+    reducers: {
+        addCategory: (state, action) => {
+            state.Categories.push(action.payload);
+        }
+
+    }   
+});
+
+export default CategoriesSlice.reducer;
```
Comment (alexander16108) [inline src/redux/categories/categoriesSlice.js]: you are to export your reducer addCategories as actions

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Eldonratzlaff  

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



## Highlights
good readme 👍🏾 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Eldonratzlaff  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 
- The website is p and running :surfer: 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**